src/pages/search/[id].vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <Head>
+    <Title>{{ APP_TITLE_SHORT }} | Search Results</Title>
+  </Head>
+
+  <page-header>
+    Search Results
+  </page-header>
+
+  <template v-if="!isLoading">
+    <app-tabs>
+      <app-tab title="Names">
+        <search-names-panel class="search__panel"/>
+      </app-tab>
+      <app-tab title="Tokens">
+        <search-tokens-panel class="search__panel"/>
+      </app-tab>
+    </app-tabs>
+
+    <app-panel class="search__panel">
+      <template #heading>
+        ACCOUNTS
+        <hint-tooltip>
+          {{ accountHints.account }}
+        </hint-tooltip>
+      </template>
+      <p class="search__paragraph">
+        No Accounts were found with the string <span class="search__highlighted">ak_-*{{ query }}</span>.
+      </p>
+      <p class="search__paragraph">
+        Please search using the complete <span class="search__highlighted">Account ID</span> to go directly to the
+        result.
+      </p>
+      <p class="search__paragraph">
+        Here are specific instructions as how to find it.
```

Comment:
I am not super satisfied with the content, maybe we can tune it during review

Reply:
It would say these instructions are incorrect. e.g. there is a transaction with id `th_yaMd8fYTRPEQdMudpuJ26J2Sx7RAz33t67Yg66epBzeBbocf9` but if you search for cf9 then you won't find it. Same for everything else except for tokens and names

Reply:
Can you provide some input here, please @michele-franchi